quotes.json

```diff
@@ -2616,6 +2608,10 @@
     "author": "George Bernard Shaw",
     "text": "The possibilities are numerous once we decide to act and not react."
   },
+  {
+    "author": "George Bernard Shaw",
+    "text": "We do not quit playing because we grow old, we grow old because we quit playing."
```

Comment:
This is one of my favourite quotes of all time. ❤️ 
I want a T-shirt with this on it. 👕 😉 
CC: @iteles / @Stephanymtr any objections to having this on the wall of the "play room" @home? 🏡